modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v2/structure.rb

```diff
@@ -82,22 +80,37 @@ def form_fill
             # NOT YET SUPPORTED
           }
 
-          fill_contestable_issues!(options)
+          fill_contestable_issues_dates!(options)
         end
         # rubocop:enable Metrics/MethodLength
 
         def insert_overlaid_pages(form_fill_path)
-          form_fill_path
+          pdftk = PdfForms.new(Settings.binaries.pdftk)
+          temp_path = "/#{::Common::FileHelpers.random_file_path}.pdf"
+          output_path = "/tmp/HLRv2-#{higher_level_review.id}-overlaid-form-fill-tmp.pdf"
+
+          Prawn::Document.generate(temp_path) do |pdf|
+            pdf.font 'Courier'
+
+            whiteout_line pdf, :first_name
+            whiteout_line pdf, :last_name
+            whiteout_line pdf, :veteran_email
+            pdf.start_new_page
+
+            # TODO: Rep first name text
+            # TODO: Rep last name text
+            whiteout_line pdf, :rep_email
+            fill_contestable_issues_text pdf
+            pdf.text_box form_data.signature,
+                         default_text_opts.merge(form_fields.boxes[:signature])
+          end
+          pdftk.multistamp(form_fill_path, temp_path, output_path)
+          output_path
         end
 
         def add_additional_pages
           return unless additional_pages?
-
-          @additional_pages_pdf ||= Prawn::Document.new(skip_page_creation: true)
-
           # TODO: Create a contestable issues table similar to NOD's extra issues table
```

Comment:
there's a PR up for this we might be able to merge and rebase.

Reply:
I believe that's still a draft PR, so I'll let that merge my changes in instead ;)